game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -26,6 +26,7 @@ const requiredState: RequiredState = {
         "item_power_treads": 1,
         "item_heart": 1,
         "item_aegis": 1,
+        "item_rapier": 1,
```

Comment:
I didn't want to add the Divine Rapier to the required state, because I wanted Slacks to tell you that he's giving you a Divine Rapier (Script_6_Closing_6). The Divine Rapier is then added afterwards.